task4.rb

```diff
@@ -3,5 +3,6 @@
 str_b = boolean.to_s
 int_b = boolean ? 1 : 0 
 
+puts "Boolean #{boolean}\nsize: 1"
```

Comment:
Hey, you have boolean size hardcoded. Please check [ObjectSpace](https://ruby-doc.org/core-2.6.1/ObjectSpace.html) and [#memsize_of](https://ruby-doc.org/stdlib-2.6.3/libdoc/objspace/rdoc/ObjectSpace.html#method-c-memsize_of)